Authenticator/Source/SegmentedControlRowCell.swift

```diff
@@ -67,8 +60,7 @@ class SegmentedControlRowCell<ViewModel: SegmentedControlRowModel>: UITableViewC
 
     // MARK: - View Model
 
-    func updateWithViewModel(viewModel: ViewModel) {
-        changeAction = viewModel.changeAction
+    func updateWithViewModel<ViewModel: SegmentedControlRowModel where ViewModel.Action == Action>(viewModel: ViewModel) {
```

Comment:
Line Length Violation: Line should be 100 characters or less: currently 122 characters (line_length)